### PR TITLE
[PL] Translation of pattern matching lesson

### PR DESCRIPTION
**Pull request by Koziolek**
in scope of issue #202

### CODE REVIEW

**Comment by doomspork** [comment]
You're a machine @Koziolek 😀 

@kdebowski could you check this one too?

File: pl/basics/pattern-matching.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+---
+layout: page
+title: Dopasowanie wzorców
+category: basics
+order: 4
+lang: pl
+---
+
+Dopasowanie wzorców to jedno z najmocniejszych elementów Elixira pozwala dopasować wartości, struktury danych, a nawet funkcje. Tę lekcję rozpoczniemy od przyjrzenia się jak używać tego mechanizmu.
```
Comment (kdebowski) [inline pl/basics/pattern-matching.md]: Something doesn't suit here very well. I think about changing the first sentence to:

> Dopasowanie wzorców, to jeden z najmocniejszych elementów Elixira, który pozwala dopasować wartości, struktury danych, a nawet funkcje.

**Comment by kdebowski** [comment]
@Koziolek, I've added a note to the first paragraph. Despite mentioned mistake everything is ok.

**Comment by Koziolek** [comment]
@doomspork [I know](http://wh40k.lexicanum.com/wiki/Fabricator-General)

@kdebowski, done. You're write. I need to read my commits one more time.

**Pull request merged**